### ISSUE TITLE
Spen integration 

### ISSUE THREAD

**Issue by ranzou06** [KotatsuApp/Kotatsu#870]
### Describe your suggested feature

Hi! It would be very good to be able to use the galaxy spen button or gestures to change page.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Koitharu** [comment]
Yes, sounds great however I cannot implement it since I have no such device so it would be nice if someone do it

**Comment by ranzou06** [comment]
Pull request opened, i just rearranged @A5101 code, TY! .
https://github.com/KotatsuApp/Kotatsu/pull/885

**Comment by ranzou06** [comment]
Pull request opened, i just implemented @A5101 code, TY! .
https://github.com/KotatsuApp/Kotatsu/pull/885

**Comment by Mathew20kdroid** [comment]
it would be awesome if you could add the **s-pen hover functionality** as well.......

Like when **hovering the s-pen over a item (Manga/Manhua)** in the **favorite tab** or while exploring Manga in **Discover tab**, it will show the **info in a popup**....... (this particular functionality can be implemented all over the app wherever there is a manga title available )
Or like in the **thumbnail preview in a manga page** ,while **hovering over the thumbnail will show a expanded or large scale view in a popup**
The possibilities are endless.......

- like hovering on a download will show detailed download progress
- detailed explanation in settings
- Reading progress bar popup in history page
- ability to open certain options while hovering over an item etc.

Really hope they add these functionality as well.........

**Comment by Koitharu** [comment]
I guess this one is already implemented, can you guys confirm if it works and we can close this?

**Comment by Mathew20kdroid** [comment]
> it would be awesome if you could add the **s-pen hover functionality** as well.......
> 
> Like when **hovering the s-pen over a item (Manga/Manhua)** in the **favorite tab** or while exploring Manga in **Discover tab**, it will show the **info in a popup**....... (this particular functionality can be implemented all over the app wherever there is a manga title available )
> Or like in the **thumbnail preview in a manga page** ,while **hovering over the thumbnail will show a expanded or large scale view in a popup**
> The possibilities are endless.......
> 
> - like hovering on a download will show detailed download progress
> - detailed explanation in settings
> - Reading progress bar popup in history page
> - ability to open certain options while hovering over an item etc.
> 
> Really hope they add these functionality as well @ranzou06 @Koitharu .........

Any chance these might be implemented???
